docs/modules/angular/nav.adoc

```diff
@@ -0,0 +1,18 @@
+* Angular
+** xref:general/index.adoc[General]
+*** xref:general/guidelines.adoc[Guidelines]
+** Cookbook
+** NgRx
+*** xref:ngrx/ngrx.adoc[Guidelines]
+*** Cookbook
+**** xref:ngrx/cookbook/addingSearchCriteria/general.adoc[Adding Search Criteria]
+***** xref:ngrx/cookbook/addingSearchCriteria/autocomplete/autocomplete.adoc[AutoComplete]
+****** xref:ngrx/cookbook/addingSearchCriteria/autocomplete/autocompleteString.adoc[AutoComplete with array of type string]
+****** xref:ngrx/cookbook/addingSearchCriteria/autocomplete/autocompleteNumber.adoc[AutoComplete with array of type number]
+****** xref:ngrx/cookbook/addingSearchCriteria/autocomplete/autocompleteObject.adoc[AutoComplete with array of type object]
+***** xref:ngrx/cookbook/addingSearchCriteria/calendar.adoc[Calendar]
+***** xref:ngrx/cookbook/addingSearchCriteria/dropdown.adoc[Dropdown]
+***** xref:ngrx/cookbook/addingSearchCriteria/multiselect.adoc[Multiselect]
+** Libraries
+*** xref:libraries/symlink.adoc[How to create a SymLink]
+*** xref:libraries/workingWithSharedLibraries.adoc[]
```

Comment:
The `docs/modules/angular/nav.adoc` is not used anymore. Add your navigation items in `docs/modules/partials/nav.adoc`